src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,10 @@
 import React from 'react';
+import { User } from '../../types/types';
 
-export const UserInfo: React.FC = () => (
+type Props = {
```

Comment:
nit: Consider renaming `list` prop to `users` for better readability and context understanding. 📖